kimai2.yml

```diff
@@ -0,0 +1,95 @@
+            MYSQL_PASSWORD: $$cap_mariadb-pass
+            MYSQL_ROOT_PASSWORD: $$cap_maridb-rootpass
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        restart: unless-stopped
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: lcxat/nginx-fpm-reverse-proxy
+        environment:
+            FPM_NGINX_HOST: srv-captain--$$cap_appname-fpm
+        volumes:
+          - $$cap_appname-public:/opt/kimai/public:ro
+        depends_on:
+          - $$cap_appname-fpm
+        restart: unless-stopped
+caproverOneClickApp:
+    variables:
+        - id: $$cap_kimai_version
+          label: Version Tag
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/kimai/kimai2/tags
+          defaultValue: 'fpm-1.15.2-prod'
+        - id: $$cap_mariadb_version
+          label: mariaDB Version
+          defaultValue: '10'
+          description: Check out their docker page for the valid tags https://hub.docker.com/_/mariadb
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_kimai-adminmail
+          label: Admin
+          description: Admin User Email
+          defaultValue: admin@admin.local
+        - id: $$cap_kimai-adminpass
+          label: Password
+          description: Admin Password
+          defaultValue: $$cap_gen_random_hex(32)
+        - id: $$cap_kimai-mailerurl
+          label: Mail URL
+          description: Mail URL, check https://www.kimai.org/documentation/emails.html for details
+          defaultValue: smtp://localhost:25?encryption=&auth_mode=
+        - id: $$cap_kimai-mailerfrom
+          lable: Mail From
+          description: an application wide “from” address for all emails
+          defaultValue: kimai@admin.local
+        - id: $$cap_mariadb-db
+          label: Database Name
+          description: Database name for kimai
+          defaultValue: 'kimai'
+        - id: $$cap_mariadb-user
+          label: mariadb User
+          description: Database User for kimai
+          defaultValue: 'kimaiuser'
+        - id: $$cap_mariadb-pass
+          label: mariadb database user password
+          description: Super secret database user password
+          defaultValue: $$cap_gen_random_hex(32)
+        - id: $$cap_mariadb-rootpass
+          label: mariadb Root password
+          description: Super secret database user password
+          defaultValue: $$cap_gen_random_hex(32)
+    instructions:
+        start: |-
+           With Kimai, the boring process of feeding Excel spreadsheets with your working hours is not only simplified, it also offers dozens of other exciting features that you don't even know you're missing so far!
+        end: |-
+            Aaaand you're done! 😄
+            Your service is available at http://$$cap_appname.$$cap_root_domain
+            -------------------------------------------------------------------
+    displayName: Kimai
+    isOfficial: true
```

Comment:
This is not the official image. It's from a 3rd party repo: https://github.com/tobybatch/kimai2
The original repo is here: https://github.com/tobybatch/kimai2

Reply:
oh yes, you are right. Kimai does refer to the third party link https://www.kimai.org/documentation/docker.html but never states it's the "official".